docs/pages/pmd/userdocs/help.md

```diff
@@ -1,8 +1,26 @@
 ---
 title: Getting Help
-sidebar: pmd_sidebar
 permalink: pmd_userdocs_help.html
-folder: pmd/userdocs
+author: Andreas Dangel <andreas.dangel@adangel.org>
+last_updated: September 2017
 ---
 
-Asking questions, reporting bugs, ...
+There are numerous ways of getting help:
+
+*   Browse already existing questions on [StackOverflow](https://stackoverflow.com/questions/tagged/pmd),
```

Comment:
shouldn't this point to https://stackoverflow.com/?

Reply:
changed